region/wreckedship/main.json

```diff
@@ -1995,7 +2009,25 @@
                     "h_canDestroyBombWalls",
                     "f_DefeatedPhantoon"
                   ],
-                  "note": "The path can't be passed unless the Workrobot is activated (so Phantoon dead)"
+                  "note": "By normal means, the path can't be passed unless the Workrobot is activated (so Phantoon dead)"
+                },
+                {
+                  "name": "WS East Supers Robot Clip (Left to Right)",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "h_canDestroyBombWalls",
+                    "canTrickyJump"
+                  ],
+                  "note": [
+                    "Kago through the Workrobot by jumping, pressing down, then morphing and moving right just before hitting the robot.",
+                    "The bomb blocks can be broken with a power bomb, morph bomb, or using screw attack.",
+                    "If using morph bombs, be careful not to break the bottom bomb tile, or you will be pushed into it and stuck."
```

Comment:
I think bomb jumping is the easier way to place the bomb at the right height.  And you unmorph before falling again.
Otherwise its a `can short jumpmorph` tech.

And the note should say a second kago will push you right into the opened morph hole.